control-plane-operator/hostedclusterconfigoperator/controllers/resources/olm/catalogs.go

```diff
@@ -31,6 +31,7 @@ func reconcileCatalogSource(cs *operatorsv1alpha1.CatalogSource, address string,
 		cs.Annotations = map[string]string{}
 	}
 	cs.Annotations["target.workload.openshift.io/management"] = `{"effect": "PreferredDuringScheduling"}`
+	cs.Annotations["openshift.io/required-scc"] = "restricted-v2"
```

Comment:
@sjenning so I tested this and I see the new annotation in the catalog source, but that doesn't seem to translate to the annotation on the catalog pod. It looks like something else is needed.